docs/DeveloperGuide.md

```diff
@@ -651,14 +655,25 @@ Use case ends.
     - 3a. Wiagi displays an error message.
       Use case restarts at step 1.
 
-## Non-Functional Requirements
+### Use Case: Getting information of commands
+**Unsure of command specifics and require assistance**
+
+**MSS**
 
-{Give non-functional requirements}
+1. User enters help into the command terminal
+2. Wiagi shows a list of all commands and their details.
+
+Use case ends.
+## Non-Functional Requirements
 
 1. A user should be alerted of the correct command format whenever an invalid command is encountered.
 2. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be 
 able to accomplish most of the tasks faster using commands than using the mouse.
 3. The system should be able to run on Windows, macOS, and Linux
+4. The system should be able to be used easily by a new user
+5. The program should not have use of language deemed offensive
+6. The program should be able to recover as much data as possible in the event of corruption
```

Comment:
Maybe specify that it is the corruption of data in storage?

Reply:
dones.